block_test.go

```diff
@@ -1601,7 +1601,9 @@ func TestTOC(t *testing.T) {
 		"#",
 		"",
 	}
-	doTestsBlock(t, tests, TOC)
+	doTestsParam(t, tests, TestParams{
+		HTMLFlags: UseXHTML | TOC,
```

Comment:
Gotcha, thanks. I didn't notice the change from `doTestsBlock` to `doTestsParam`.